games/f.yaml

```diff
@@ -434,10 +448,9 @@
   development: active
   originals:
   - Warlords II
-  repo: https://gitlab.com/freedroid/freedroid-src
   type: remake
   updated: 2015-04-06
-  url: https://sourceforge.net/p/freelords/wiki/Home/
+  repo: https://sourceforge.net/p/freelords/wiki/Home/
```

Comment:
this looks more like a url; yes it's on sourceforge but it doesn't show source control information on this page
for repo https://sourceforge.net/projects/freelords/ is fine